[OpenWrt 24.10] kernel: bridge: readding MLD wakeup call feature

T-X opened the pull request:
With the update to OpenWrt 24.10 the bridge MLD wakeup call feature was removed. However the issue with Android devices and MLD is still present. Therefore readding the bridge MLD wakeup call patch and porting it to Linux 6.6 / OpenWrt 24.10.

Link: https://issuetracker.google.com/issues/149630944
Fixes: b7481fd08e44 ("modules: switch to OpenWrt 24.10")

herbetom commented:
Now that #3370 has been merged this PR should be switched to target main and if necessary rebased. Thanks!:)

T-X commented:
Changelog v2:
* rebased to new, updated main branch after #3370 was merged -> no conflicts, minor update through Gluon "make refresh-patches"
* updated referenced commit ID in the "Fixes:" line
